packages/date/src/index.js

```diff
@@ -470,58 +610,8 @@ export function isInTheFuture( dateValue ) {
  * @return {Date} Date
  */
 export function getDate( dateString ) {
-	if ( ! dateString ) {
-		return momentLib.tz( WP_ZONE ).toDate();
-	}
-
-	return momentLib.tz( dateString, WP_ZONE ).toDate();
-}
-
-/**
- * Creates a moment instance using the given timezone or, if none is provided, using global settings.
- *
- * @param {Date|string|Moment|null} dateValue Date object or string, parsable
- *                                            by moment.js.
- * @param {string|number|null}      timezone  Timezone to output result in or a
- *                                            UTC offset. Defaults to timezone from
- *                                            site.
- *
- * @see https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
- * @see https://en.wikipedia.org/wiki/ISO_8601#Time_offsets_from_UTC
- *
- * @return {Moment} a moment instance.
- */
-function buildMoment( dateValue, timezone = '' ) {
-	const dateMoment = momentLib( dateValue );
-
-	if ( timezone && ! isUTCOffset( timezone ) ) {
-		return dateMoment.tz( timezone );
-	}
-
-	if ( timezone && isUTCOffset( timezone ) ) {
-		return dateMoment.utcOffset( timezone );
-	}
-
-	if ( settings.timezone.string ) {
-		return dateMoment.tz( settings.timezone.string );
-	}
-
-	return dateMoment.utcOffset( settings.timezone.offset );
-}
-
-/**
- * Returns whether a certain UTC offset is valid or not.
- *
- * @param {number|string} offset a UTC offset.
- *
- * @return {boolean} whether a certain UTC offset is valid or not.
- */
-function isUTCOffset( offset ) {
-	if ( 'number' === typeof offset ) {
-		return true;
-	}
-
-	return VALID_UTC_OFFSET.test( offset );
+	const actualDate = dateString ? new Date( dateString ) : new Date();
+	return toDate( actualDate, { timeZone: getActualTimezone() } );
 }
 
-setupWPTimezone();
+export { zonedTimeToUtc };
```

Comment:
What makes us need to export zonedTimeToUtc?

Reply:
Only the one call that you mentioned bellow; I will remove this export once we decide which way to go with using timezones for I18n dates.